src/components/common/articleCard/ratingComponent/index.js

```diff
@@ -0,0 +1,26 @@
+import React, { Component } from 'react';
+import Rating from 'react-rating';
+import style from './ratingComponent.scss';
+
+const theme = {
+  color: '#ffd700',
+  border: '1px solid #fff',
+  fontSize: '13px',
+};
+export default class RatingArticleCardComponent extends Component {
```

Comment:
Rating component should be put directly in the common since it will be reused

Reply:
Alright.

Reply:
This has been fixed in the landing page PR. We are using the same articleCard component.